.github/workflows/test-runner-on-perf.yml

```diff
@@ -71,12 +71,12 @@ jobs:
           fi
       - name: "Whitelist Runner IP"
         run: |
-          CUR_IPS=$(gcloud container clusters describe ${K8_CLUSTER} --format json | \
+          CUR_IPS=$(gcloud container clusters describe ${K8_CLUSTER} --zone ${GOOGLE_ZONE} --format json | \
```

Comment:
Not sure it would help here but there is a way to change the default zone for a google project locally, using gcloud or environment variables. https://cloud.google.com/compute/docs/gcloud-compute#set_default_region_and_zone_in_your_local_client